t/plugin/google-cloud-logging.t

```diff
@@ -313,6 +313,8 @@ qr/\{\"error\"\:\"[\w+\s+]*\"\}/
 --- error_log
 Batch Processor[google-cloud-logging] failed to process entries
 Batch Processor[google-cloud-logging] exceeded the max_retry_count
+--- skip_openssl
```

Comment:
We need to add comment about why this test is skipped under OpenSSL 3.0. If possible, we can modify the test to make it pass under different OpenSSL versions.

Reply:
openssl 3.0 with fips enabled requires keys larger than 2048 bits. All tests skipped for openssl 3.0 use small key.

Reply:
Those tests just aims for small keys.

Reply:
Are `test route (no access to this scopes)` in google-cloud-logging.t and `use lifetime_grace_period default value` in jwt-auth2.t (and other similar tests) tests for small keys?

Reply:
We can give a new key in those tests.